### ISSUE TITLE
PNI ordering of news updates doesn't work in the CMS

### ISSUE THREAD

**Issue by caltrider** [mozilla/foundation.mozilla.org#7448]
When I try to reorder the news updates with up/down arrows in the CMS they don't seem to save and reorder. They move around but once I save, they seem to revert back to the order they were before. 

### To Reproduce
1. Go to any PNI product page in the CMS. Go down to the product updates section near the bottom.
2. Click on the up or down arrows to reorder a news updates
3. Scroll down save draft. 
4. Now go down and see if they updates are reordered in the order you'd hoped. They keep jumping around or not reordering for me. 

### Expected behavior
The updates reorder into the order I placed them and stay that way once saved and published.

**Comment by Pomax** [comment]
This appears to be a two parter:
1. we forgot to add the `Orderable.Meta` to the product update meta class, which is kind of necessary (either no meta class, or make sure the meta class extends `Orderable.Meta`). This fixes the "saving" part of the problem
2. even with the `Orderable.Meta` superclass, and an explicit `ordering = ['sort_order']`, the actual ordering in terms of templating does not seem to change, which means that even if the ordering is correct in the CMS, that order doesn't make it onto the actual page users look at.

**Comment by Pomax** [comment]
nope, I am an idiot, of course the live page won't update if you only save a draft.
